ostd/src/arch/x86/irq.rs

```diff
@@ -104,14 +104,23 @@ impl IrqLine {
         if has_interrupt_remapping() {
             let handle = alloc_irt_entry();
             if let Some(handle) = handle {
+                // Enable irt entry
+                handle
+                    .lock()
+                    .irt_entry_mut()
+                    .unwrap()
+                    .enable_default(irq_num as u32);
                 irq.bind_remapping_entry.call_once(|| handle);
             }
         }
         irq
     }
 
-    pub fn bind_remapping_entry(&self) -> Option<&Arc<SpinLock<IrtEntryHandle, LocalIrqDisabled>>> {
-        self.bind_remapping_entry.get()
+    /// Gets the remapping index of the IRQ line.
+    ///
+    /// Return None if architecture does not support interrupt remapping.
```

Comment:
- `None` -> `` `None` ``
 - Or the interrupt remapping is not enabled?
```suggestion
    /// This method will return `None` if interrupt remapping is disabled or
    /// not supported by the architecture.
```